src/lexedata/enrich/segment_using_clts.py

```diff
@@ -27,6 +27,16 @@ def cleanup(form: str) -> str:
 
 
 def segment_form(form: str) -> t.Iterable[pyclts.models.Symbol]:
```

Comment:
I have been working with some other transcription stuff in the last few days and I have made my version of the segmenter more robust, in particular with respect to pre-nasalization and pre-aspiration. I may drop it in here at some point. #